docs/configuration.md

```diff
@@ -409,6 +413,12 @@ The target location to which Lighthouse CI should upload the reports.
 - You want to control access to your Lighthouse reports.
 - You've setup a [Lighthouse CI server](./server.md).
 
+**When to use `target=filesystem`:**
+
+- You want to process the raw Lighthouse results yourself locally.
+- You want access to the report files on the local filesystem.
+- You don't want to upload the results to a custom location that isn't supported by Lighthouse CI.
```

Comment:
Looks like an incorrect double-negative here. You probably meant that you _do_ want to upload results to a location _not_ supported, or vice versa.

Reply:
thanks @alexfornuto!